string type
geometry_msgs/Pose pose
geometry_msgs/Polygon geometry
# z coordinate will be stripped
